app/views/projects/samples/attachments/_table.html.erb

```diff
@@ -6,7 +6,9 @@
   >
     <thead class="text-slate-700 bg-slate-50 dark:bg-slate-900 dark:text-slate-400">
       <tr>
-        <th aria-hidden="true" class="px-6 py-3"></th>
+        <% if allowed_to?(:update?, @project) %>
```

Comment:
Same comment as above to use `update_sample?`

Reply:
Updated in e75c67e5f4998e09d5e36342368a13419a860e91